// @esri/calcite-components/dist/cjs/bs-e182722e.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v2.8.3
 */
'use strict';

const calciteInputTimePicker_calciteTimePicker_entry = require('./calcite-input-time-picker.calcite-time-picker-96222bfc.js');
require('./index-edc00aa2.js');
require('./form-3d45b7d3.js');
require('./dom-d6c2a698.js');
require('./guid-4b381af5.js');
require('./resources-2b0eb58d.js');
require('./interactive-8ff4eb13.js');
require('./browser-23ebed5a.js');
require('./key-2ea701e3.js');
require('./label-933b1f63.js');
require('./component-c82b66a6.js');
require('./loadable-52c88e74.js');
require('./locale-801208c5.js');
require('./observers-7d0e25b9.js');
require('./focusTrapComponent-2f50fda2.js');
require('./math-d3d12bfd.js');
require('./t9n-9d0bfc6a.js');
require('./openCloseComponent-8bbb13df.js');
require('./Validation-e0f4c1ec.js');

// Bosnian [bs]
var locale = {
  name: 'bs',
  weekdays: 'nedjelja_ponedjeljak_utorak_srijeda_četvrtak_petak_subota'.split('_'),
  months: 'januar_februar_mart_april_maj_juni_juli_august_septembar_oktobar_novembar_decembar'.split('_'),
  weekStart: 1,
  weekdaysShort: 'ned._pon._uto._sri._čet._pet._sub.'.split('_'),
  monthsShort: 'jan._feb._mar._apr._maj._jun._jul._aug._sep._okt._nov._dec.'.split('_'),
  weekdaysMin: 'ne_po_ut_sr_če_pe_su'.split('_'),
  ordinal: function ordinal(n) {
    return n;
  },
  formats: {
    LT: 'H:mm',
    LTS: 'H:mm:ss',
    L: 'DD.MM.YYYY',
    LL: 'D. MMMM YYYY',
    LLL: 'D. MMMM YYYY H:mm',
    LLLL: 'dddd, D. MMMM YYYY H:mm'
  }
};
calciteInputTimePicker_calciteTimePicker_entry.dayjs.locale(locale, null, true);

exports['default'] = locale;
