// @esri/calcite-components/dist/cjs/nl-f25baed8.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v2.8.3
 */
'use strict';

const calciteInputTimePicker_calciteTimePicker_entry = require('./calcite-input-time-picker.calcite-time-picker-96222bfc.js');
require('./index-edc00aa2.js');
require('./form-3d45b7d3.js');
require('./dom-d6c2a698.js');
require('./guid-4b381af5.js');
require('./resources-2b0eb58d.js');
require('./interactive-8ff4eb13.js');
require('./browser-23ebed5a.js');
require('./key-2ea701e3.js');
require('./label-933b1f63.js');
require('./component-c82b66a6.js');
require('./loadable-52c88e74.js');
require('./locale-801208c5.js');
require('./observers-7d0e25b9.js');
require('./focusTrapComponent-2f50fda2.js');
require('./math-d3d12bfd.js');
require('./t9n-9d0bfc6a.js');
require('./openCloseComponent-8bbb13df.js');
require('./Validation-e0f4c1ec.js');

// Dutch [nl]
var locale = {
  name: 'nl',
  weekdays: 'zondag_maandag_dinsdag_woensdag_donderdag_vrijdag_zaterdag'.split('_'),
  weekdaysShort: 'zo._ma._di._wo._do._vr._za.'.split('_'),
  weekdaysMin: 'zo_ma_di_wo_do_vr_za'.split('_'),
  months: 'januari_februari_maart_april_mei_juni_juli_augustus_september_oktober_november_december'.split('_'),
  monthsShort: 'jan_feb_mrt_apr_mei_jun_jul_aug_sep_okt_nov_dec'.split('_'),
  ordinal: function ordinal(n) {
    return "[" + n + (n === 1 || n === 8 || n >= 20 ? 'ste' : 'de') + "]";
  },
  weekStart: 1,
  yearStart: 4,
  formats: {
    LT: 'HH:mm',
    LTS: 'HH:mm:ss',
    L: 'DD-MM-YYYY',
    LL: 'D MMMM YYYY',
    LLL: 'D MMMM YYYY HH:mm',
    LLLL: 'dddd D MMMM YYYY HH:mm'
  },
  relativeTime: {
    future: 'over %s',
    past: '%s geleden',
    s: 'een paar seconden',
    m: 'een minuut',
    mm: '%d minuten',
    h: 'een uur',
    hh: '%d uur',
    d: 'een dag',
    dd: '%d dagen',
    M: 'een maand',
    MM: '%d maanden',
    y: 'een jaar',
    yy: '%d jaar'
  }
};
calciteInputTimePicker_calciteTimePicker_entry.dayjs.locale(locale, null, true);

exports['default'] = locale;
