// @esri/calcite-components/dist/esm/da-7ee02d6d.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v2.8.3
 */
import { d as dayjs } from './calcite-input-time-picker.calcite-time-picker-e20d2a85.js';
import './index-aad43fe9.js';
import './form-f71acf93.js';
import './dom-f20a2cef.js';
import './guid-61099ad8.js';
import './resources-283ff61b.js';
import './interactive-8dda3b79.js';
import './browser-789c6078.js';
import './key-22771b72.js';
import './label-3cec3979.js';
import './component-154b01bb.js';
import './loadable-f0718ca4.js';
import './locale-fc68a30a.js';
import './observers-372e145d.js';
import './focusTrapComponent-57b40800.js';
import './math-80b4cae8.js';
import './t9n-afcafb6c.js';
import './openCloseComponent-ef904cc3.js';
import './Validation-1b8ccecb.js';

// Danish [da]
var locale = {
  name: 'da',
  weekdays: 'søndag_mandag_tirsdag_onsdag_torsdag_fredag_lørdag'.split('_'),
  weekdaysShort: 'søn._man._tirs._ons._tors._fre._lør.'.split('_'),
  weekdaysMin: 'sø._ma._ti._on._to._fr._lø.'.split('_'),
  months: 'januar_februar_marts_april_maj_juni_juli_august_september_oktober_november_december'.split('_'),
  monthsShort: 'jan._feb._mar._apr._maj_juni_juli_aug._sept._okt._nov._dec.'.split('_'),
  weekStart: 1,
  ordinal: function ordinal(n) {
    return n + ".";
  },
  formats: {
    LT: 'HH:mm',
    LTS: 'HH:mm:ss',
    L: 'DD.MM.YYYY',
    LL: 'D. MMMM YYYY',
    LLL: 'D. MMMM YYYY HH:mm',
    LLLL: 'dddd [d.] D. MMMM YYYY [kl.] HH:mm'
  },
  relativeTime: {
    future: 'om %s',
    past: '%s siden',
    s: 'få sekunder',
    m: 'et minut',
    mm: '%d minutter',
    h: 'en time',
    hh: '%d timer',
    d: 'en dag',
    dd: '%d dage',
    M: 'en måned',
    MM: '%d måneder',
    y: 'et år',
    yy: '%d år'
  }
};
dayjs.locale(locale, null, true);

export { locale as default };
