// @esri/calcite-components/dist/esm/en-ca-2878caa5.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v2.8.3
 */
import { d as dayjs } from './calcite-input-time-picker.calcite-time-picker-e20d2a85.js';
import './index-aad43fe9.js';
import './form-f71acf93.js';
import './dom-f20a2cef.js';
import './guid-61099ad8.js';
import './resources-283ff61b.js';
import './interactive-8dda3b79.js';
import './browser-789c6078.js';
import './key-22771b72.js';
import './label-3cec3979.js';
import './component-154b01bb.js';
import './loadable-f0718ca4.js';
import './locale-fc68a30a.js';
import './observers-372e145d.js';
import './focusTrapComponent-57b40800.js';
import './math-80b4cae8.js';
import './t9n-afcafb6c.js';
import './openCloseComponent-ef904cc3.js';
import './Validation-1b8ccecb.js';

// English (Canada) [en-ca]
var locale = {
  name: 'en-ca',
  weekdays: 'Sunday_Monday_Tuesday_Wednesday_Thursday_Friday_Saturday'.split('_'),
  months: 'January_February_March_April_May_June_July_August_September_October_November_December'.split('_'),
  weekdaysShort: 'Sun_Mon_Tue_Wed_Thu_Fri_Sat'.split('_'),
  monthsShort: 'Jan_Feb_Mar_Apr_May_Jun_Jul_Aug_Sep_Oct_Nov_Dec'.split('_'),
  weekdaysMin: 'Su_Mo_Tu_We_Th_Fr_Sa'.split('_'),
  ordinal: function ordinal(n) {
    return n;
  },
  formats: {
    LT: 'h:mm A',
    LTS: 'h:mm:ss A',
    L: 'YYYY-MM-DD',
    LL: 'MMMM D, YYYY',
    LLL: 'MMMM D, YYYY h:mm A',
    LLLL: 'dddd, MMMM D, YYYY h:mm A'
  },
  relativeTime: {
    future: 'in %s',
    past: '%s ago',
    s: 'a few seconds',
    m: 'a minute',
    mm: '%d minutes',
    h: 'an hour',
    hh: '%d hours',
    d: 'a day',
    dd: '%d days',
    M: 'a month',
    MM: '%d months',
    y: 'a year',
    yy: '%d years'
  }
};
dayjs.locale(locale, null, true);

export { locale as default };
