// @esri/calcite-components/dist/esm/it-7e07efae.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v2.8.3
 */
import { d as dayjs } from './calcite-input-time-picker.calcite-time-picker-e20d2a85.js';
import './index-aad43fe9.js';
import './form-f71acf93.js';
import './dom-f20a2cef.js';
import './guid-61099ad8.js';
import './resources-283ff61b.js';
import './interactive-8dda3b79.js';
import './browser-789c6078.js';
import './key-22771b72.js';
import './label-3cec3979.js';
import './component-154b01bb.js';
import './loadable-f0718ca4.js';
import './locale-fc68a30a.js';
import './observers-372e145d.js';
import './focusTrapComponent-57b40800.js';
import './math-80b4cae8.js';
import './t9n-afcafb6c.js';
import './openCloseComponent-ef904cc3.js';
import './Validation-1b8ccecb.js';

// Italian [it]
var locale = {
  name: 'it',
  weekdays: 'domenica_lunedì_martedì_mercoledì_giovedì_venerdì_sabato'.split('_'),
  weekdaysShort: 'dom_lun_mar_mer_gio_ven_sab'.split('_'),
  weekdaysMin: 'do_lu_ma_me_gi_ve_sa'.split('_'),
  months: 'gennaio_febbraio_marzo_aprile_maggio_giugno_luglio_agosto_settembre_ottobre_novembre_dicembre'.split('_'),
  weekStart: 1,
  monthsShort: 'gen_feb_mar_apr_mag_giu_lug_ago_set_ott_nov_dic'.split('_'),
  formats: {
    LT: 'HH:mm',
    LTS: 'HH:mm:ss',
    L: 'DD/MM/YYYY',
    LL: 'D MMMM YYYY',
    LLL: 'D MMMM YYYY HH:mm',
    LLLL: 'dddd D MMMM YYYY HH:mm'
  },
  relativeTime: {
    future: 'tra %s',
    past: '%s fa',
    s: 'qualche secondo',
    m: 'un minuto',
    mm: '%d minuti',
    h: 'un\' ora',
    hh: '%d ore',
    d: 'un giorno',
    dd: '%d giorni',
    M: 'un mese',
    MM: '%d mesi',
    y: 'un anno',
    yy: '%d anni'
  },
  ordinal: function ordinal(n) {
    return n + "\xBA";
  }
};
dayjs.locale(locale, null, true);

export { locale as default };
