// @esri/calcite-components/dist/esm/sv-4f2659e2.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v2.8.3
 */
import { d as dayjs } from './calcite-input-time-picker.calcite-time-picker-e20d2a85.js';
import './index-aad43fe9.js';
import './form-f71acf93.js';
import './dom-f20a2cef.js';
import './guid-61099ad8.js';
import './resources-283ff61b.js';
import './interactive-8dda3b79.js';
import './browser-789c6078.js';
import './key-22771b72.js';
import './label-3cec3979.js';
import './component-154b01bb.js';
import './loadable-f0718ca4.js';
import './locale-fc68a30a.js';
import './observers-372e145d.js';
import './focusTrapComponent-57b40800.js';
import './math-80b4cae8.js';
import './t9n-afcafb6c.js';
import './openCloseComponent-ef904cc3.js';
import './Validation-1b8ccecb.js';

// Swedish [sv]
var locale = {
  name: 'sv',
  weekdays: 'söndag_måndag_tisdag_onsdag_torsdag_fredag_lördag'.split('_'),
  weekdaysShort: 'sön_mån_tis_ons_tor_fre_lör'.split('_'),
  weekdaysMin: 'sö_må_ti_on_to_fr_lö'.split('_'),
  months: 'januari_februari_mars_april_maj_juni_juli_augusti_september_oktober_november_december'.split('_'),
  monthsShort: 'jan_feb_mar_apr_maj_jun_jul_aug_sep_okt_nov_dec'.split('_'),
  weekStart: 1,
  yearStart: 4,
  ordinal: function ordinal(n) {
    var b = n % 10;
    var o = b === 1 || b === 2 ? 'a' : 'e';
    return "[" + n + o + "]";
  },
  formats: {
    LT: 'HH:mm',
    LTS: 'HH:mm:ss',
    L: 'YYYY-MM-DD',
    LL: 'D MMMM YYYY',
    LLL: 'D MMMM YYYY [kl.] HH:mm',
    LLLL: 'dddd D MMMM YYYY [kl.] HH:mm',
    lll: 'D MMM YYYY HH:mm',
    llll: 'ddd D MMM YYYY HH:mm'
  },
  relativeTime: {
    future: 'om %s',
    past: 'för %s sedan',
    s: 'några sekunder',
    m: 'en minut',
    mm: '%d minuter',
    h: 'en timme',
    hh: '%d timmar',
    d: 'en dag',
    dd: '%d dagar',
    M: 'en månad',
    MM: '%d månader',
    y: 'ett år',
    yy: '%d år'
  }
};
dayjs.locale(locale, null, true);

export { locale as default };
