// @esri/calcite-components/dist/esm/tr-ff2c8ebc.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v2.8.3
 */
import { d as dayjs } from './calcite-input-time-picker.calcite-time-picker-e20d2a85.js';
import './index-aad43fe9.js';
import './form-f71acf93.js';
import './dom-f20a2cef.js';
import './guid-61099ad8.js';
import './resources-283ff61b.js';
import './interactive-8dda3b79.js';
import './browser-789c6078.js';
import './key-22771b72.js';
import './label-3cec3979.js';
import './component-154b01bb.js';
import './loadable-f0718ca4.js';
import './locale-fc68a30a.js';
import './observers-372e145d.js';
import './focusTrapComponent-57b40800.js';
import './math-80b4cae8.js';
import './t9n-afcafb6c.js';
import './openCloseComponent-ef904cc3.js';
import './Validation-1b8ccecb.js';

// Turkish [tr]
var locale = {
  name: 'tr',
  weekdays: 'Pazar_Pazartesi_Salı_Çarşamba_Perşembe_Cuma_Cumartesi'.split('_'),
  weekdaysShort: 'Paz_Pts_Sal_Çar_Per_Cum_Cts'.split('_'),
  weekdaysMin: 'Pz_Pt_Sa_Ça_Pe_Cu_Ct'.split('_'),
  months: 'Ocak_Şubat_Mart_Nisan_Mayıs_Haziran_Temmuz_Ağustos_Eylül_Ekim_Kasım_Aralık'.split('_'),
  monthsShort: 'Oca_Şub_Mar_Nis_May_Haz_Tem_Ağu_Eyl_Eki_Kas_Ara'.split('_'),
  weekStart: 1,
  formats: {
    LT: 'HH:mm',
    LTS: 'HH:mm:ss',
    L: 'DD.MM.YYYY',
    LL: 'D MMMM YYYY',
    LLL: 'D MMMM YYYY HH:mm',
    LLLL: 'dddd, D MMMM YYYY HH:mm'
  },
  relativeTime: {
    future: '%s sonra',
    past: '%s önce',
    s: 'birkaç saniye',
    m: 'bir dakika',
    mm: '%d dakika',
    h: 'bir saat',
    hh: '%d saat',
    d: 'bir gün',
    dd: '%d gün',
    M: 'bir ay',
    MM: '%d ay',
    y: 'bir yıl',
    yy: '%d yıl'
  },
  ordinal: function ordinal(n) {
    return n + ".";
  }
};
dayjs.locale(locale, null, true);

export { locale as default };
